specs/experimental/gov-proposal-validator.md

```diff
@@ -51,67 +51,157 @@ The contract has a single `owner` role (Optimism Foundation) with permissions to
+the proposal needs to reach to pass or the number of top choices that can pass the voting.
 
-- The user MUST use the `submitFundingProposal` that uses specific `calldata` pre-defined by the owner
+For the `ProposalOptions` of the voting module, these are:
+- `string[] descriptions`: The strings of the different options that can be voted.
 
-Note: `MaintenanceUpgradeProposals` type can move straight to voting if all submission checks pass, unlike the rest of the
-proposals where they need to collect a number of approvals by top delegates in order to move to vote. This call should be
+`submitMaintenanceUpgradeProposal`
+
+Submits a Maintenance Upgrade proposal to move for voting. `MaintenanceUpgradeProposals` type can move
+straight to voting if all submission checks pass, unlike the rest of the proposals where they
+need to collect a number of approvals by top delegates in order to move to vote. This call should be
 atomic.
 
+- MUST be called by an approved address
+- MUST check if the proposal is a duplicate
+- MUST use the `Optimistic` Voting Module
+- MUST provide a valid attestation UID
+- MUST NOT do any operations
+- MUST emit `ProposalSubmitted` and `ProposalVotingModuleData` events
+- MUST store submission proposal data which are defined by the `ProposalSubmissionData` struct
+
 ```solidity
-function submitProposal(
-    address[] memory _targets,
-    uint256[] memory _values,
-    bytes[] memory _calldatas,
+function submitMaintenanceUpgradeProposal(
+    uint248 _againstThreshold,
+    bool _isRelativeToVotableSupply,
     string memory _description,
     ProposalType _proposalType,
     bytes32 _attestationUid
 ) external returns (bytes32 proposalHash_);
 ```
 
+Optimistic Voting Module
+Maintenance Upgrade proposals use the `Optimistic` voting module.
+This requires the user who submits the proposal to provide some additional data related to the proposal.
+
+For the `ProposalSettings` of the voting module, these are:
+- `uint248 againstThreshold`: The threshold of the against option.
+- `bool isRelativeToVotableSupply`: True if the against threshold should be relative to the votable supply.
+
+`submitCouncilMemberElectionsProposal`
+
+Submits a Council Member Elections proposal for approval and voting.
+
+- MUST be called by an approved address
+- MUST check if the proposal is a duplicate
+- MUST use the `Ranked Choice` Voting Module
+- MUST provide a valid attestation UID
+- MUST NOT do any operations
+- MUST emit `ProposalSubmitted` and `ProposalVotingModuleData` events
+- MUST store submission proposal data which are defined by the `ProposalSubmissionData` struct
+
+```solidity
+function submitCouncilMemberElectionsProposal(
+    uint128 _criteriaValue,
+    string[] _optionDescriptions,
+    string memory _description,
+    ProposalType _proposalType,
+    bytes32 _attestationUid
+) external returns (bytes32 proposalHash_);
+```
+
+Approval Voting Module
```

Comment:
In the google sheets with the proposal details they had shared this proposal type should be using a "Ranked Choice" voting module, however seems like they use the `Approval` Voting Module with "TopChoices" as a criteria type. Again not 100% sure on this need confirmation from agora.